client/src/main/scala/sbtcompendium/client/CompendiumClient.scala

```diff
@@ -102,7 +106,17 @@ object CompendiumClient {
         } yield out
       }
 
-      override def generateClient(target: IdlName, identifier: String): F[String] = F.pure("")
+      override def generateClient(target: IdlName, identifier: String): F[List[String]] =
+        target match {
+          case IdlName.Avro =>
+            retrieveProtocol(identifier, None)
+              .map(_.map(r => handleAvro(r.raw)).getOrElse(List.empty))
+          case _ =>
+            Sync[F].raiseError(UnknownError(s"Unknown error with status code 501. Schema format not implemented yet"))
```

Comment:
With context-applied, this could be just:

```suggestion
            F.raiseError(UnknownError(s"Unknown error with status code 501. Schema format not implemented yet"))
```

Reply:
Another option (since we have the `cats.implicits._` import) is just
```suggestion
            UnknownError(s"Unknown error with status code 501. Schema format not implemented yet").raiseError[F, List[String]]
```

Reply:
the ```
UnknownError(s"Unknown error with status code 501. Schema format not implemented yet")
              .raiseError[F, List[String]]
```  option looks cool 😎